catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/model/Search.java

```diff
@@ -184,9 +184,9 @@ private boolean isSuccessful(final Set<ProcessingDetails> details) {
 
     private void updateResultStatus(List<Result> results) {
         Bag hitSourceCount = new HashBag();
-
-        for (String sourceId : queryStatus.keySet()) {
-            queryStatus.get(sourceId).setResultCount(0);
+        
+        for (Map.Entry<String, QueryStatus> entry : queryStatus.entrySet()) {
```

Comment:
If we're not using the key at all in this block, could replace the `entrySet()` with `values()`.